tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -38,16 +45,19 @@ def activate(self, context: str) -> Mapping[str, Any]:
 
     @command
     def capture_screenshot(
-        self,
-        context: str,
-        clip: Optional[ClipOptions] = None,
-        origin: Optional[OriginOptions] = None,
-    ) -> Mapping[str, Any]:
+            self,
```

Comment:
I know that this is indentation again but we use 4 chars or align it by the opening bracket the line before. This (8 chars) is unusual, and does flake8 complain?